benchmark/scala/src/main/scala/com/microsoft/tpch/ConvertTpchCsvToParquetApp.scala

```diff
@@ -0,0 +1,137 @@
+/*
+ * Licensed to the .NET Foundation under one or more agreements.
+ * The .NET Foundation licenses this file to you under the MIT license.
+ * See the LICENSE file in the project root for more information.
+ */
+
+package mypackage
+
+import org.apache.spark.sql.types._
+import org.apache.spark.sql.{SaveMode, SparkSession}
+
+object ConvertTpchCsvToParquetApp {
+  def main(args: Array[String]): Unit = {
+    val spark = SparkSession.builder()
```

Comment:
something like:
```scala
if (args.length != 2) {
  println("Usage:")
  println("\t<spark-submit> --master local[*] --class com.microsoft.tpch.ConvertTpchCsvToParquetApp microsoft-spark-benchmark-<version>.jar")
  println("\t\t<input-directory-with-TPCH-tables> <output-directory>")
  return
}
```
?

Reply:
That is a good idea, args checking is always needed actually. Thanks!

A small modification on the printout, just following the format of [submitting application](https://spark.apache.org/docs/latest/submitting-applications.html). The instruction for submitting spark for this conversion application will be the same as the following printout.

`    
    if (args.length != 2) {
      println(
        "Error: incorrect command to submit the application, please follow the instruction and specify the path to the source and destination directory as arguments."
      )
      println("The correct command usage should be:")
      println("\t./bin/spark-submit \\")
      println("\t\t--class com.microsoft.tpch.ConvertTpchCsvToParquetApp \\")
      println("\t\t--master local[*] \\")
      println("\t\tmicrosoft-spark-benchmark-<version>.jar \\")
      println(
        "\t\tpath-to-source-directory-with-TPCH-tables path-to-destination-directory-to-save-parquet-file"
      )
      return
    }
`